### PR TITLE
Fix packs with demisto-sdk lint issues

### PR DESCRIPTION
**Pull request by ilaner**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR should fix packs which were broken as a result of broken dependencies in the test image that `lint` uses.
Should be merged with this PR:
https://github.com/demisto/demisto-sdk/pull/3116
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Integrations/CoreIOCs/CoreIOCs_test.py [hunk 1]
```diff
@@ -70,7 +70,6 @@ def test_http_request_bad_json(self, requests_mock):
         assert e.value.message == f'Could not parse json out of {text}'
         assert e.value.res.status_code == 200
         assert isinstance(e.value.exception, json.JSONDecodeError)
-        assert e.value.exception.args == ('Expecting value', 'not a json')
```
Comment (dorschw) [inline Packs/Core/Integrations/CoreIOCs/CoreIOCs_test.py:73]: Can we replace it with another check instead of removing?

Comment (ilaner) [reply Packs/Core/Integrations/CoreIOCs/CoreIOCs_test.py:73]: the behavior is tested in line 70 (the message includes `not a json` message)

**Review by dorschw — APPROVED** [review]

File: .gitlab/ci/on-push.yml [hunk 2]
```diff
@@ -587,7 +587,15 @@ test-upload-flow:
...
-    - export pipeline_id=$(./Utils/trigger_test_upload_flow.sh -ct $GITLAB_SVC_USER_TOKEN -g -b $BRANCH | jq .id)
+    # retry mechanism for trigger upload pipeline in case it failed because of gitlab connectivity issues.
+    - for _ in {1..3}; do
+    -   export pipeline_id=$(./Utils/trigger_test_upload_flow.sh -ct $GITLAB_SVC_USER_TOKEN -g -b $BRANCH | jq .id)
+    -   if [ "${pipeline_id}" != "null" ]; then
+    -     break
+    -   fi
+    -   echo "Sleeping for 10 seconds before retrying"
+    -   sleep 10
+    - done
```
Comment (daryakoval) [inline .gitlab/ci/on-push.yml:598]: Adi added it, strange it is in this pr...

Comment (ilaner) [reply .gitlab/ci/on-push.yml]: updated the branch

**Pull request merged**